pareto/utilities/results.py

```diff
@@ -158,31 +158,43 @@ def generate_report(model, is_print=[], fname=None):
 
```

Comment:
General question regarding the results: I've noticed that - for the generic case - most of the produced water now ends up going to disposal (over 88%)... Why is that? Did building pipelines become more expensive? Just seems a bit high and certainly does not align with current industry best practices. Might be worth making some revisions to the data to promote higher reuse rates.

Reply:
Markus, we are going to address this in a future PR, for which we will include the calculation of levelized capital costs for pipelines, this will decrease the cost of a pipeline and promote building pipeline infrastructure to increase water reuse